### PR TITLE
Fix remainder of incorrect electrical contact nomenclature

### PR DESCRIPTION
**Pull request by sapitts**
Corrects the nomenclature corrections not made in #22270 as identified by @cticenhour 

Refs #22114

### CODE REVIEW

File: modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ In base SI units this Lagrange multiplier variable has the units $\left[ \frac{A
 where $J$ is the current density, $\sigma$ is the electrical conductivity, and $E$ is the electric field.
 
 
-!alert note title=Employ Consistency in Primary and Secondary Designations
-Consistency in the selection of the primary boundary and secondary boundary among the electrical, thermal, and interface Joule Heating mortar contact input file compoments is recommended.
+!alert note title=Closed Gap Interface Assumed by this Class
+The `ADInterfaceJouleHeatingConstraint` class should only be employed in simulations when the user is certain that the current density like electric potential contact Lagrange multiplier variable is nonzero only when the interface gap is closed. The `ADInterfaceJouleHeatingConstraint` class may also be used in simulations with an open gap at the interface, so long as the electric potential contact Lagrange multiplier variable across that gap is zero while the interface gap is open.
```
Comment (cticenhour) [inline modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md:41]:
```suggestion
The `ADInterfaceJouleHeatingConstraint` class should only be employed in simulations when the user is certain that the current-density-like electric potential contact Lagrange multiplier variable is nonzero only when the interface gap is closed. The `ADInterfaceJouleHeatingConstraint` class may also be used in simulations with an open gap at the interface, so long as the electric potential contact Lagrange multiplier variable across that gap is zero while the interface gap is open.
```
Should use hyphens here when the phrase is used as a single adjective modifying "electric potential".

File: modules/heat_conduction/test/tests/interface_heating_mortar/constraint_joule_heating_single_material_insulated.i [hunk 2]
```diff
@@ -5,7 +5,7 @@
 # page for ADInterfaceJouleHeatingConstraint for the relevant formulas).
 #
 # With the prescribed 0.0 V (left) and 0.3V (right) potential boundary conditions, the
-# electric potential flux is 9.25e6 [V * S / m^2]. From this electric potential flux,
+# electric potential drop is 9.25e6 [V * S / m^2]. From this current density like LM variable,
```
Comment (cticenhour) [inline modules/heat_conduction/test/tests/interface_heating_mortar/constraint_joule_heating_single_material_insulated.i:8]:
```suggestion
# electric potential drop is 9.25e6 [V * S / m^2]. From this current-density-like LM variable,
```

**Review by cticenhour — CHANGES_REQUESTED** [review]

File: modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md (hunk 1 above)
Comment (sapitts) [reply modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md:41]: I think you've become an NS&T tech editor!

Comment (cticenhour) [reply modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md:41]: It's a blessing and a curse 😂

**Review by cticenhour — APPROVED** [review]

**Pull request merged**